packages/circuit-ui/components/Table/components/TableHeader/TableHeader.tsx

```diff
@@ -161,7 +162,8 @@ const condensedColStyles = ({
   condensed &&
   scope === 'col' &&
   css`
-    padding: ${theme.spacings.byte} ${theme.spacings.mega};
+    padding: ${theme.spacings.kilo} ${theme.spacings.mega}
+      ${theme.spacings.kilo} ${theme.spacings.giga};
```

Comment:
The vertical spacing was `byte` before, I assume this was an accidental change? 

```suggestion
    padding: ${theme.spacings.byte} ${theme.spacings.mega}
      ${theme.spacings.byte} ${theme.spacings.giga};
```

Reply:
Thanks @connor-baer for spotting this! It was indeed an accidental change! I have fixed it, committed, added a changeset for this patch. Let me know if there's something else I missed out on!